app/builders/work_builder/work_type_builder.rb

```diff
@@ -16,7 +16,7 @@ def call
     # @return [Cocina::Models::DescriptiveValue] descriptive value for Stanford self-deposit resource types
     def self_deposit_form
       @self_deposit_form ||= cocina_object.description.form.find do |form|
-        form&.source&.value == ToCocina::Work::TypesMapper::RESOURCE_TYPE_SOURCE_LABEL
+        form&.source&.value == WorkMapper::TypesMapper::RESOURCE_TYPE_SOURCE_LABEL
```

Comment:
Now I'm totally confused what's a mapper and what's a builder.

Reply:
A builder builds a form object. A mapper maps to Cocina. Very arbitrary and wholly changeable!